arch/arm/core/cortex_m/mpu/arm_mpu.c

```diff
@@ -459,9 +437,11 @@ static void _arm_mpu_config(void)
 			mpu_config.num_regions,
 			_get_num_regions()
 		);
-		return;
+		return -1;
```

Comment:
Isn't it better to use some standard Zephyr error code here, e.g. -EINVAL or some other?

Reply:
I don't see the point as this gets called by init code which just checks success/failure

Reply:
That's right